hadoop-project/pom.xml

```diff
@@ -185,7 +185,7 @@
     <make-maven-plugin.version>1.0-beta-1</make-maven-plugin.version>
     <surefire.fork.timeout>900</surefire.fork.timeout>
     <aws-java-sdk.version>1.12.262</aws-java-sdk.version>
-    <hsqldb.version>2.3.4</hsqldb.version>
+    <hsqldb.version>2.5.2</hsqldb.version>
```

Comment:
We probably need to do this separately.

Reply:
Thank you very much for helping to review the code!

I have submitted a separate pr for upgrading hsqldb version, can you help to review it? 

HADOOP-18451. Update hsqldb.version from 2.3.4 to 2.5.2. 
PR: https://github.com/apache/hadoop/pull/4880